srcpkgs/odrive/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'odrive'
+pkgname=odrive
+version=0.2.0
+revision=1
+create_wrksrc=yes
+only_for_archs="x86_64"
+short_desc="Google Drive GUI for linux"
+maintainer="Jaredy <jaredycalves@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/liberodark/ODrive"
+distfiles="https://github.com/liberodark/ODrive/releases/download/${version}/odrive_${version}_amd64.deb"
```

Comment:
get from source code and compile it if it's free software

Reply:
Done